### PR TITLE
fix(TrendChart): Tooltip shows units for hours and GB

### LINKED ISSUE #245: Overview - All Cards: Trend Lines Tooltips needs units of measurement
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by boaz0**
fixes #245

### CODE REVIEW

File: src/utils/formatValue.ts [hunk 1]
```diff
@@ -56,5 +66,5 @@ export const formatStorage: ValueFormatter = (
   _unit,
   { fractionDigits = 2 } = {}
 ) => {
-  return value.toFixed(fractionDigits);
+  return `${unknownTypeFormatter(value, _unit, { fractionDigits })} GB`;
```
Comment (dlabrecq) [inline src/utils/formatValue.ts]: Do we want to hard code `hours` and `GB` here? Perhaps the component should add the localized labels and leave this util to just format the value itself?

Comment (boaz1337) [reply src/utils/formatValue.ts]: From my understanding `ValueFormatter` should do the localized labels.
I might have a better way to format the labels without creating a new function for every new unit.

**Comment by boaz1337** [comment]
ping

File: src/utils/formatValue.ts (hunk 1 above)
Comment (dlabrecq) [reply src/utils/formatValue.ts]: Are you still planning to  localize labels like 'GB' and 'Hours'?

**Comment by boaz1337** [comment]
ping? @dlabrecq

**Comment by dlabrecq** [comment]
I had asked (above) if you're still planning to localize the 'GB' and 'Hours' labels?

**Comment by boaz1337** [comment]
Sorry didn't see that.

File: src/utils/formatValue.ts (hunk 1 above)
Comment (boaz1337) [reply src/utils/formatValue.ts]: Yes sure. I am working on it..

**Pull request merged**